reporter/sources/php/execution_timeouts.py

```diff
@@ -0,0 +1,51 @@
+from reporter.helpers import is_production_host
+from reporter.reports import Report
+from common import PHPLogsSource
+
+
+class PHPExecutionTimeoutSource(PHPLogsSource):
+    """
+    Source of PHP execution timeouts.
+    Since this problem is commonly caused by extremely large articles,
+    reports should be sent to Community Support.
+    """
+
+    REPORT_LABEL = "PHP Execution Timeout"
```

Comment:
Let's create tickets that have this label directly in ComSup project.

The code that can do that - https://github.com/Wikia/jira-reporter/blob/master/reporter/classifier/classifier.py#L57-L70